### PR TITLE
Release 4.0.0

### PR DESCRIPTION
**Pull request by voxel01**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,29 @@ All notable changes to this project will be documented in this file.
...
 
-## [v3.0.0](https://github.com/voxpupuli/puppet-varnish/tree/v3.0.0) (2023-02-28)
+## [v3.1.0](https://github.com/voxpupuli/puppet-varnish/tree/v3.1.0) (2023-05-12)
 
-[Full Changelog](https://github.com/voxpupuli/puppet-varnish/compare/d55e143663f24b4f5efd8a9628a3d0173264609b...v3.0.0)
+[Full Changelog](https://github.com/voxpupuli/puppet-varnish/compare/v3.0.0...v3.1.0)
+
+**Breaking changes:**
+
+- Drop Puppet 6 support [\#20](https://github.com/voxpupuli/puppet-varnish/pull/20) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (bastelfreak) [inline CHANGELOG.md:13]: the release needs to be a major one (4.0.0) because of this PR

Comment (smortex) [inline CHANGELOG.md:13]: According to semver, breaking changes require to bump the major version, so this should be 4.0.0.

Comment (voxel01) [reply CHANGELOG.md:13]: Thanks, did major instead of minor

Comment (voxel01) [reply CHANGELOG.md:13]: Thanks, did major instead of minor

**Review by smortex — APPROVED** [review]
LGTM!

**Comment by smortex** [comment]
@voxel01 I fixed CI in #21 that is now merged.  Can you rebase on top of master and regenerate the changelog?

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**